lib/parsers/jpeg_parser.rb

```diff
@@ -22,6 +22,7 @@ def call(io)
     @width             = nil
     @height            = nil
     @exif_data         = nil
+    @exif_data_frames = []
```

Comment:
Can you please, please, pretty please add one more space :-)

Reply:
Totally. And remove `@exif_data` which we no longer need